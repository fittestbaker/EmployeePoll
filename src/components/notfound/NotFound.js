const NotFound = () => {
  return (
    <div className="NotFound">
      <h1>404 NotFound</h1>
    </div>
  );
};

export default NotFound;
