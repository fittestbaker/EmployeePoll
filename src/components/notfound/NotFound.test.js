import { render, screen } from "@testing-library/react";
import NotFound from "./NotFound";
import { MemoryRouter } from "react-router";

describe("render NotFound in document", () => {
  it("should be successful", async () => {
    render(
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>
    );

    const button = screen.getByText(/NotFound/i);
    expect(button).toBeInTheDocument();
  });
});

describe("render 404 in document", () => {
  it("should be successful", async () => {
    render(
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>
    );

    const button = screen.getByText(/404/i);
    expect(button).toBeInTheDocument();
  });
});

describe("render NotFound snapshot", () => {
  it("should be successful", async () => {
    var { component } = render(
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>
    );

    expect(component).toMatchSnapshot();
  });
});
