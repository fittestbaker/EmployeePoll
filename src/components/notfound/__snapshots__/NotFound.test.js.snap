// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NotFound snapshot should be successful 1`] = `undefined`;
